### DISCUSSION TITLE
[Question] Do you have an example of envoy?

### DISCUSSION THREAD

**Discussion by eottabom** [testcontainers/testcontainers-java#8772] [Q&A]
You can use envoy as a side-car or edge.

envoy has a lot of filter.

However, it is expected that there will be many difficulties in testing it in a local environment.

So it would be nice to have an example through testcontainer, but don't you have one?

**Comment by eottabom** [comment]
I don't think it's in the example, can I work on it briefly?

For example, you can use it like a service mesh

**Reply by eddumelendez** [reply]
Hi, please feel free to submit an example

**Reply by eottabom** [reply]
@eddumelendez Better late than never! 😄

It took a while, but I've finally opened a PR for the Envoy service mesh example: #12081

It covers an Envoy sidecar and xDS setup. Would love your feedback when you have time!
